.gitignore

```diff
@@ -1,7 +1,6 @@
 .chalice/deployments/
 .chalice/deployed/
 .chalice/venv/
-.chalice/config.json
```

Comment:
So this is now a checked-in file?

Reply:
Yeah, I'm not sure why it wasn't, it seems pretty static; it does not seem to deployed, i.e. seems like only used for running locally (and in fact, though the file needs to exist, if it just contains {} it seems okay) - I should ask Will about this.